tests/acceptance/features/bootstrap/WebDavLockingContext.php

```diff
@@ -574,6 +654,25 @@ public function numberOfLockShouldBeReported($count, $file, $user) {
 		);
 	}
 
+	/**
+	 * @Then /^the (?:files|folders|resources) should contain the following locks:$/
+	 *
+	 * @param TableNode $lockTable
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function numberOfTheFollowingLockShouldBeReported(TableNode $lockTable):void {
+		$this->featureContext->verifyTableNodeColumns($lockTable, ["user", "resource", "number0fLock"]);
```

Comment:
```suggestion
		$this->featureContext->verifyTableNodeColumns($lockTable, ["user", "resource", "locks_count"]);
```
is that a `0` or `O`?